src/styles/global.css

```diff
@@ -80,4 +80,8 @@
     src: url('/src/fonts/DMMono-Regular.ttf') format('truetype');
     font-weight: normal;
     font-style: normal;
+}
+
+html{
```

Comment:
Do not use overflow like this, If You facing overflow issue then find the reason of overflow and solve it there.